tests/locator_test.go

```diff
@@ -206,3 +206,24 @@ func TestLocatorFocus(t *testing.T) {
 		require.Panics(t, func() { link.Focus(nil) }, "should not select multiple elements")
 	})
 }
+
+// Skip adding t.Parallel to subtests because they might cause flakiness.
```

Comment:
In this case I'm pretty sure there would be a consistent data race either in goja or in our own code, so not really flakiness.

Reply:
Please see: 9fd4fc1